recipes/mirotalk-c2c/package.json

```diff
@@ -0,0 +1,12 @@
+{
+	"id": "mirotalk-c2c",
+	"name": "MiroTalk C2C",
+	"version": "1.0.1",
+	"license": "MIT",
+	"config": {
+		"serviceURL": "https://c2c.mirotalk.com",
+    "hasCustomUrl": true,
```

Comment:
Can this service be self-hosted? Is that why the `hasCustomUrl` is `true`? If not, please remove it

Reply:
Yes, All MiroTalk versions can be self hosted, so I want to keep "hasCustomUrl": true, to allow anyone to change the default URL with they own.